tensorboard/plugins/beholder/demos/demo/demo.py

```diff
@@ -166,9 +171,9 @@ def feed_dict(train):
 
   for i in range(FLAGS.max_steps):
     # if i % 10 == 0:  # Record summaries and test-set accuracy
-        summary, acc = sess.run([merged, accuracy], feed_dict=feed_dict(False))
-        test_writer.add_summary(summary, i)
-        print('Accuracy at step %s: %s' % (i, acc))
+    summary, acc = sess.run([merged, accuracy], feed_dict=feed_dict(False))
```

Comment:
I think these were indented to match the commented-out if-else-else structure surrounding them.  Maybe we should just remove those comments entirely?  I suspect they're a holdover from the original demo file.

Reply:
Indeed. Done!